### PR TITLE
Skip should set offline cpus after deploy PAO test

### PR DESCRIPTION
**Pull request by marioferh**
Test should be skipped if onlineCPUS is less than 8.

Sometimes when the nodes where the test is executed have 4 CPUs, and we set offline 2 to check the normal behavior, the test are so slow and timeout is reach. 

To avoid this situation, we check onlineCPUs are enough.

Signed-off-by: Mario Fernandez <mariofer@redhat.com>

### CODE REVIEW

File: test/e2e/performanceprofile/functests/2_performance_update/updating_profile.go
Comment (mrniranjan) [inline test/e2e/performanceprofile/functests/2_performance_update/updating_profile.go:719]: should we add skip out of the for loop, not sure if skip breaks the loop ?

Comment (marioferh) [reply test/e2e/performanceprofile/functests/2_performance_update/updating_profile.go:719]: "Skip notifies Ginkgo that the current spec was skipped." I think so

File: test/e2e/performanceprofile/functests/2_performance_update/updating_profile.go
Comment (marioferh) [reply test/e2e/performanceprofile/functests/2_performance_update/updating_profile.go]: right

**Review by fromanirh — APPROVED** [review]
LGTM.
Please get a formal LGTM tag from QE folks.

**Comment by fromanirh** [comment]
/hold
please unhold once LGTM'd

File: test/e2e/performanceprofile/functests/2_performance_update/updating_profile.go
Comment (shajmakh) [inline test/e2e/performanceprofile/functests/2_performance_update/updating_profile.go:716]: since it is int, better to have the int format as %d. not sure if %s will still reflect valid value. was this tested?

Comment (marioferh) [reply test/e2e/performanceprofile/functests/2_performance_update/updating_profile.go:716]: printing the string: onlineCPUCount
int is onlineCPUInt. It's no so clear sorry

Comment (shajmakh) [reply test/e2e/performanceprofile/functests/2_performance_update/updating_profile.go:716]: oh I misread that, thanks for pointing this out

**Pull request merged**
